ext/v8/enums.h

```diff
@@ -0,0 +1,27 @@
+#include "rr.h"
+
+namespace rr {
+
+  class Enums {
+  public:
+    static void Init() {
+      ClassBuilder("AccessControl").
+        defineConst("DEFAULT", INT2FIX(v8::DEFAULT)).
+        defineConst("ALL_CAN_READ", INT2FIX(v8::ALL_CAN_READ)).
+        defineConst("ALL_CAN_WRITE", INT2FIX(v8::ALL_CAN_WRITE)).
+        defineConst("PROHIBITS_OVERWRITING", INT2FIX(v8::PROHIBITS_OVERWRITING));
+
+      ClassBuilder("PropertyAttribute").
+        defineConst("None", INT2FIX(v8::None)).
+        defineConst("ReadOnly", INT2FIX(v8::ReadOnly)).
+        defineConst("DontEnum", INT2FIX(v8::DontEnum)).
+        defineConst("DontDelete", INT2FIX(v8::DontDelete));
+    }
+
+    template <class T>
+    static T fromRubyValue(VALUE rb_int) {
+      return (T)NUM2INT(rb_int);
```

Comment:
Were you seeing a warning with this? I was just using `NUM2INT()` and it was working fine. I think what would be nice is a constructor that takes a `VALUE` and a default to pass in the case that the value is null:

```C++
template <class T>
class Enum {
  Enum(VALUE value_, int default_) : value(value_), default(default_) {}

  operator T() {
    return RTEST(value) ? NUM2INT(value_) : NUM2INT(default);
  }
}
```
That way, we can use it like `Enum<v8::PropertyAttribute>(attribute, v8::None);`